app/controllers/people_controller.rb

```diff
@@ -0,0 +1,19 @@
+class PeopleController < ApplicationController
+  def index
+    @people = Person.all
+
+  end
+
+  def show
+    @person = Person.find(params[:person_id])
+
+    render :person
+  end
+
+  def tasks_for_person
+    @person = Person.find(params[:person_id])
+    @tasks = Task.where(person_id: @person.id)
```

Comment:
I think you could just use `@person.tasks` to retrieve these rather than going back to the Task model